Makefile

```diff
@@ -37,6 +37,19 @@ evmone:
 	cmake -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_SHARED_LIBRARY_SUFFIX_CXX=.so ; \
 	cmake --build build --parallel -t evmone
 
+coverage-go: TOSCA_GO_COVERAGE_DIR=./go/build/coverage_report/${TOSCA_GO_COVERAGE_EVM}
+coverage-go:
+	go build -cover -o ${TOSCA_GO_COVERAGE_DIR}/driver ./go/ct/driver/ ; \
+	GOCOVERDIR=${TOSCA_GO_COVERAGE_DIR} ${TOSCA_GO_COVERAGE_DIR}/driver run --max-errors 1 ${TOSCA_GO_COVERAGE_EVM} ; \
+	go tool covdata textfmt --i ${TOSCA_GO_COVERAGE_DIR} -o ${TOSCA_GO_COVERAGE_DIR}/driver_coverage_report.txt ; \
+	go tool cover -html ${TOSCA_GO_COVERAGE_DIR}/driver_coverage_report.txt -o ${TOSCA_GO_COVERAGE_DIR}/coverage_output.html
+
+coverage-lfvm: TOSCA_GO_COVERAGE_EVM=lfvm
+coverage-lfvm: coverage-go
+
+coverage-geth: TOSCA_GO_COVERAGE_EVM=geth
+coverage-geth: coverage-go
```

Comment:
nit: should those targets get a "ct-" prefix. Something like `ct-coverage-geth` and `ct-coverage-lfvm`?